docs/static/releasenotes.asciidoc

```diff
@@ -48,18 +48,19 @@ It can automatically delete segments from a dead letter queue after all events i
 
 Several AWS plugins are now bundled in a single {logstash-ref}/plugins-integrations-aws.html[AWS integration plugin], enabling easier maintenance and upgrades of AWS-based
   plugins. They all use version 3 of the AWS Ruby SDK.
+  
 [[jdk-8.4.0]]
 ===== JDK17 support
 
-Logstash now comes bundled with JDK17, while still providing compatibility with user-supplied JDK11.
+Logstash now comes bundled with JDK17, while still providing compatibility with user-supplied JDK11. 
+The JDK update pertains to a potential security vulnerability. 
```

Comment:
```suggestion
The new JDK includes an update to a potential security vulnerability. 
```

Reply:
Do you think we still need `pertain(s|ing) to` in the description? Rather than updating the vulnerability? 
```suggestion
The new JDK includes an update pertaining to a potential security vulnerability.
```